docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
++  to: oadp-operator-1.4
+ - dockerfile_path: hack/ci.Dockerfile
+   from: src
+-  to: test-oadp-operator
++  to: test-oadp-operator-1.4
+ operator:
+   bundles:
+   - dockerfile_path: bundle.Dockerfile
+   substitutions:
+-  - pullspec: quay.io/konveyor/oadp-operator:latest
+-    with: oadp-operator
++  - pullspec: quay.io/konveyor/oadp-operator:oadp-1.4
++    with: oadp-operator-1.4
+ releases:
+...
+     env:
+-      OO_CHANNEL: stable
++      OO_CHANNEL: stable-1.4
+       OO_INSTALL_NAMESPACE: openshift-adp
+...
+         namespace: test-credentials
+-      from: test-oadp-operator
++      from: test-oadp-operator-1.4
+       resources:
+...
+     env:
+-      OO_CHANNEL: stable
++      OO_CHANNEL: stable-1.4
+       OO_INSTALL_NAMESPACE: openshift-adp
+...
+         namespace: test-credentials
+-      from: test-oadp-operator
++      from: test-oadp-operator-1.4
+       resources:
+...
+     env:
+-      OO_CHANNEL: stable
++      OO_CHANNEL: stable-1.4
+       OO_INSTALL_NAMESPACE: openshift-adp
+...
+         namespace: test-credentials
+-      from: test-oadp-operator
++      from: test-oadp-operator-1.4
+       resources:
+...
+ zz_generated_metadata:
+-  branch: master
++  branch: oadp-1.4
+   org: openshift
+   repo: oadp-operator
+...
+```
+
+> **Note**: to get diff between files, you can run `diff -ruN diff -ruN ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.VERSION.yaml ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.VERSION.yaml`.
+
+After creating these files, run `make jobs`.
+
+Finally, add image to `core-services/image-mirroring/konveyor/mapping_konveyor_latest`
+```diff
+...
+ registry.ci.openshift.org/konveyor/oadp-operator:oadp-operator-1.3 quay.io/konveyor/oadp-operator:oadp-1.3-latest quay.io/konveyor/oadp-operator:oadp-1.3-amd64 quay.io/konveyor/oadp-operator:oadp-1.3
++registry.ci.openshift.org/konveyor/oadp-operator:oadp-operator-1.4 quay.io/konveyor/oadp-operator:oadp-1.4-latest quay.io/konveyor/oadp-operator:oadp-1.4-amd64 quay.io/konveyor/oadp-operator:oadp-1.4
+...
+```
+
+#### Example: updating release branch for OADP operator repository
+
+For this example, lets say new release branch is `oadp-1.4`.
+
+Update `Makefile`, changing the following (if release branch was created from master branch).
+```diff
+...
+-DEFAULT_VERSION := 99.0.0
++DEFAULT_VERSION := 1.4.0
+...
+-CHANNELS = "stable"
++CHANNELS = "stable-1.4"
+...
+-DEFAULT_CHANNEL = "stable"
++DEFAULT_CHANNEL = "stable-1.4"
+...
+-IMG ?= quay.io/konveyor/oadp-operator:latest
++IMG ?= quay.io/konveyor/oadp-operator:oadp-1.4
+...
+```
+
+> **Note**: after creating release branch, update `Makefile` in master branch, changing the following.
+>```diff
+> # A valid Git branch from https://github.com/openshift/oadp-operator
+>-PREVIOUS_CHANNEL ?= oadp-1.3
+>+PREVIOUS_CHANNEL ?= oadp-1.4
+> # Go version in go.mod in that branch
+>-PREVIOUS_CHANNEL_GO_VERSION ?= 1.20
+>+PREVIOUS_CHANNEL_GO_VERSION ?= 1.22
```

Comment:
Do we help here the developer to find out what go versions are available for a particular OCP or OADP? It's easy to follow the documentation, but it's harder to define which version is correct.

Reply:
developer needs to check `go.mod` of previous version